### ISSUE TITLE
App crashes continuously after passcode activity 

### ISSUE THREAD

**Issue by PatelVatsalB21** [openMF/android-client#1629]
**Summary:** 
Sometimes randomly app crashes after passcode activity with evernote jobs error that it cannot schedule more than 100 distinct jobs and then continuosly crashes in every launch and user cannot go ahead of passcode activity. This error does not occur frequently but might be a severe one as app needs re-install after this error. Can we replace **Evernote jobs** with [Work Manager](https://developer.android.com/topic/libraries/architecture/workmanager) ? 

**Screenshots:** 
![AppScheduleMoreThan100error](https://user-images.githubusercontent.com/70195106/101981634-6c9d3300-3c94-11eb-91ef-612e84f9cce3.png)

**Device and Android version:** 
Lenovo Tab 7 
Android 7

**Comment by PatelVatsalB21** [comment]
@robustTechie have look at this error , I would like to work on this error if we migrate to Work Manager.

**Comment by danishjamal104** [comment]
Yes @PatelVatsalB21 , I have also faced the same issue but think it should be handled in passcode module rather than app. I haven't explored it much because once I faced this issue but clearing app from RAM and relaunching remove the issue. So @robustTechie please look into this and suggest the changes to @PatelVatsalB21 he will fix this.

**Comment by PatelVatsalB21** [comment]
> Yes @PatelVatsalB21 , I have also faced the same issue but think it should be handled in passcode module rather than app. I haven't explored it much because once I faced this issue but clearing app from RAM and relaunching removed the issue. So @robustTechie please look into this and suggest the changes to @PatelVatsalB21 he will fix this.

It's possible that on higher android and hardware based model it might be an RAM based issue but for my specified devise it is much frequent almost occurs every 15-20 launches and need to be reinstalled. If we extend support till API version 15 this might be bigger problem for smaller devices

**Comment by danishjamal104** [comment]
Which API level you faced this issuee??

**Comment by PatelVatsalB21** [comment]
I have problem on android 7.0 (API lvl 24)

**Comment by robustTechie** [comment]
Hey @PatelVatsalB21 as for shifting to work manager may solve this issue but we need to discuss over it once :)

**Comment by NikunjGarg2000** [comment]
Yes @PatelVatsalB21, @danishjamal104 , @robustTechie I am facing this error continuously in Realme X running on Android 10. I have to run project and reinstall app every single time I want to open it. Without running it crashes every time.

**Comment by therajanmaurya** [comment]
No longer a issue
